src/components/userComponents/Footer/Footer.tsx

```diff
@@ -0,0 +1,557 @@
+              href="https://www.facebook.com/PHSSPCA/"
+              className="w-[27px] h-[27px] justify-center items-center flex"
+            >
+              {' '}
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                x="0px"
+                y="0px"
+                width="50"
+                height="50"
+                viewBox="0 0 50 50"
+                className="fill-current text-white active:text-[#bcc0bb]"
+              >
+                <path d="M25,3C12.85,3,3,12.85,3,25c0,11.03,8.125,20.137,18.712,21.728V30.831h-5.443v-5.783h5.443v-3.848 c0-6.371,3.104-9.168,8.399-9.168c2.536,0,3.877,0.188,4.512,0.274v5.048h-3.612c-2.248,0-3.033,2.131-3.033,4.533v3.161h6.588 l-0.894,5.783h-5.694v15.944C38.716,45.318,47,36.137,47,25C47,12.85,37.15,3,25,3z" />
+              </svg>
+            </Link>
+            <Link
+              href="https://www.instagram.com/peninsulahumanesociety"
+              className="w-[27px] h-[27px] justify-center items-center flex"
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                x="0px"
+                y="0px"
+                width="30"
+                height="30"
+                viewBox="0 0 30 30"
+                className="fill-current text-white active:text-[#bcc0bb]"
+              >
+                <path d="M 9.9980469 3 C 6.1390469 3 3 6.1419531 3 10.001953 L 3 20.001953 C 3 23.860953 6.1419531 27 10.001953 27 L 20.001953 27 C 23.860953 27 27 23.858047 27 19.998047 L 27 9.9980469 C 27 6.1390469 23.858047 3 19.998047 3 L 9.9980469 3 z M 22 7 C 22.552 7 23 7.448 23 8 C 23 8.552 22.552 9 22 9 C 21.448 9 21 8.552 21 8 C 21 7.448 21.448 7 22 7 z M 15 9 C 18.309 9 21 11.691 21 15 C 21 18.309 18.309 21 15 21 C 11.691 21 9 18.309 9 15 C 9 11.691 11.691 9 15 9 z M 15 11 A 4 4 0 0 0 11 15 A 4 4 0 0 0 15 19 A 4 4 0 0 0 19 15 A 4 4 0 0 0 15 11 z" />
+              </svg>
+            </Link>
+            <Link
+              href="https://twitter.com/peninsulahumane"
+              className="w-[27px] h-[27px] justify-center items-center flex"
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                x="0px"
+                y="0px"
+                width="30"
+                height="30"
+                viewBox="0 0 30 30"
+                className="fill-current text-white active:text-[#bcc0bb]"
+              >
+                <path d="M26.37,26l-8.795-12.822l0.015,0.012L25.52,4h-2.65l-6.46,7.48L11.28,4H4.33l8.211,11.971L12.54,15.97L3.88,26h2.65 l7.182-8.322L19.42,26H26.37z M10.23,6l12.34,18h-2.1L8.12,6H10.23z" />
+              </svg>
+            </Link>
+            <Link
+              href="https://www.tiktok.com/@peninsulahumanesociety"
+              className="w-[27px] h-[27px] justify-center items-center flex"
+            >
+              <svg
+                xmlns="http://www.w3.org/2000/svg"
+                x="0px"
+                y="0px"
```

Comment:
@jxmoose  is there a reason we use svgs and drawn paths instead of react icons?
Would cut down on writing + more readable this way.
https://react-icons.github.io/react-icons/search/#q=Twitter